### PR TITLE
[session02] #115 한민영 EDA & 전처리 과제, 통계 사전과제 제출합니다.

### PR DESCRIPTION
**Pull request by hmy9691-jpg**
### 🔍 어려웠던 점 및 새로 알게 된 점 간단히 정리
Scaler 활용하는 게 어려웠습니다.

### 🔍 질문 및 하고 싶은 말
통계 부분 다 배웠다고 생각했는데 다시 정리하니 헷갈리는 내용도 많아서 다음 시간 집중해서 잘 들어야겠어요💪

---
### ✅ PR Checklist
PR이 다음 요구 사항을 충족하는지 확인하세요.
- [x] 본인 브랜치 → develop 브랜치 로 PR 합니다.

### CODE REVIEW

**Comment by jjang-yukyung** [comment]
일단 마크다운 사전 과제를 너무너무 잘해줬어요!!!!
거의 마크다운 교본 수준 ㄷㄷ

Scaler 활용이 어렵다고 했는데 사실 이것만 기억해주면 됩니다
1) 학습용 데이터에서는 .fit_transform() 쓰기
2) 평가용 데이터에서는 .transform()만 쓰기!
그리고 숫자 배열만 보면 감이 잘 안오니까 스케일링 전후 분포를 seaborn의 hisplot으로 시각화해보면 감이 올거예요. X축의 눈금 범위가 어떻게 변하는지 보면 스케일러 전후가 확실히 비교되거덩요

1. assignment_eda_1.ipynb
- 판다스 데이터 정말 잘 불러왔고 seaborn의 barplot 등을 활용해서 코드도 잘 작성해주셨어용
- 그리고 그래프 바탕으로 생존율이 가장 높은 등급은 1등급이라는 점도 정확하게 잘 도출했습니다!

2. assignment_eda_2.ipynb
- 결측치를 전체 평균이나 최빈값으로 일괄 채우지 않고 determind_kids, determine_composition 함수를 만들어서 axis=1로 추정해 결측치를 채운 것도 너무 잘해줬어요!
- 추가적으로 1인 가구 결측치를 대체할 때 np.random.choice([3, 4], p=[0.5, 0.5])로 50% 확률 값을 할당해줬는데 모델의 재현성을 위해서는 코드 상단에 np.random.seed()를 고정해주는것도 추천해요!!

전반적으로 너무 깔끔하게 잘한 과제!!!!!! 인사이콘은 식은죽먹기일듯합니다 ♥️👍
이번 세션도 화이팅!
